plugins/modules/mysql_db.py

```diff
@@ -316,7 +335,10 @@
 
 from ansible.module_utils.basic import AnsibleModule
 from ansible_collections.community.mysql.plugins.module_utils.database import mysql_quote_identifier
-from ansible_collections.community.mysql.plugins.module_utils.mysql import mysql_connect, mysql_driver, mysql_driver_fail_msg, mysql_common_argument_spec
+from ansible_collections.community.mysql.plugins.module_utils.mysql import (
```

Comment:
```suggestion
from ansible_collections.community.mysql.plugins.module_utils.mysql import (
```
please don't mix several unrelated changes in one PR as it makes things harder to review. Could you please eliminate all unrelated changes (like format fixes)?

Reply:
yes